hedera-mirror-rest/utils.js

```diff
@@ -660,6 +660,41 @@ const parsePublicKeyQueryParam = (parsedQueryParams, columnName) => {
   );
 };
 
+/**
+ * If enabled in config, ensure the returned timestamp range is fully bound; contains both a begin
+ * and end timestamp value. The provided Range is not modified. If changes are made a copy is returned.
+ *
+ * @param {Range} tsRange timestamp range, typically based on query parameters
+ * @return {Range} fully bound timestamp range
+ */
+const bindTimestampRange = function (tsRange) {
```

Comment:
we generally use arrow function

```suggestion
const bindTimestampRange = (tsRange) => {
```

Reply:
👍

Reply:
Done